BackofficeBundle/Tests/EventSubscriber/DataTransformer/ValueTransformerManagerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Tests\EventSubscriber\DataTransformer;
+
+use OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer\ValueTransformerManager;
+use Phake;
+
+/**
+ * Class ContentTypeTypeSubscriberTest
+ */
+class ContentTypeTypeSubscriberTest extends \PHPUnit_Framework_TestCase
+{
+    protected $manager;
+    protected $valueTransformer;
+    /**
+     * Set up the test
+     */
+    public function setUp()
+    {
+        $this->manager = new ValueTransformerManager();
+        $this->valueTransformer = Phake::mock('OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer\ValueTransformerInterface');
+        Phake::when($this->valueTransformer)->getName()->thenReturn('1');
+        Phake::when($this->valueTransformer)->support(Phake::anyParameters())->thenReturn(true);
```

Comment:
you should modify the support return in the test, this way you can add multiple value transformer, and see which one is used